oct/pages/registration.py

```diff
@@ -1,28 +1,69 @@
-from oct.pages import not_implemented
-from oct.pages.base import Page
+from typing import Any
 from selenium.webdriver import Remote
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.common.by import By
+from oct.pages.base import Page
+from oct.pages import not_implemented
 
 
 class PersonalDetails:
-    def type_first_name(self, first_name: str) -> None:
+    def type_first_name(self, first_name: str, browser: Remote) -> str:
+        wait = WebDriverWait(browser, 10)
```

Comment:
@AlekseyDidilov `until` return instance of found WebElement.
```suggestion
        first_name_field = WebDriverWait(browser, 10).until(
               EC.visibility_of_element_located((By.ID, "input-firstname"))
       )
```